internal/uvm/memory_update.go

```diff
@@ -0,0 +1,46 @@
+package uvm
+
+import (
+	"context"
+	"fmt"
+
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+)
+
+const (
+	bytesPerPage = 4096
+	bytesPerMB   = 1024 * 1024
+)
+
+// UpdateMemory makes a call to the VM's orchestrator to update the VM's size in MB
+// Internally, HCS will get the number of pages this corresponds to and attempt to assign
+// pages to numa nodes evenly
```

Comment:
We don't have the plumbing for NUMA support in this repo yet.

Reply:
Might add a note regarding that here

Reply:
Just leave this for now as discussed offline. We'll likely forget to add it back whenever vNUMA might get in

Reply:
Talked offline, but I don't think we should add a note about that here. This was meant to just describe the behavior of the implementation. If we end up adding vnuma support later, we may forget to update any note made here and cause confusion.